buildSrc/src/main/kotlin/deps/AppDependencies.kt

```diff
@@ -0,0 +1,60 @@
+package deps
+
+object AppDependencies {
+
+  object Androidx {
+    const val activity = "androidx.activity:activity:${versions.AppDependencies.Androidx.activity}"
+    const val appCompat = "androidx.appcompat:appcompat:${versions.AppDependencies.Androidx.appCompat}"
+    const val constraintLayout = "androidx.constraintlayout:constraintlayout:${versions.AppDependencies.Androidx.constraintLayout}"
+    const val fragment = "androidx.fragment:fragment-ktx:${versions.AppDependencies.Androidx.fragment}"
+    const val desugar = "com.android.tools:desugar_jdk_libs:${versions.AppDependencies.Androidx.desugar}"
+    const val materialDesign = "com.google.android.material:material:${versions.AppDependencies.Androidx.materialDesign}"
+    const val recyclerView = "androidx.recyclerview:recyclerview:${versions.AppDependencies.Androidx.recyclerView}"
+    const val work = "androidx.work:work-runtime-ktx:${versions.AppDependencies.Androidx.work}"
+
+    object Room {
+      const val compiler = "androidx.room:room-compiler:${versions.AppDependencies.Androidx.room}"
+      const val ktx = "androidx.room:room-ktx:${versions.AppDependencies.Androidx.room}"
+      const val runtime = "androidx.room:room-runtime:${versions.AppDependencies.Androidx.room}"
+    }
+  }
+
+  object Cql {
+    const val cqlEngineCore = "org.opencds.cqf:cql-engine:${versions.AppDependencies.Cql.cqlEngine}"
+    const val cqlEngineFhir = "org.opencds.cqf:cql-engine-fhir:${versions.AppDependencies.Cql.cqlEngine}"
+    const val hapiR4 = "ca.uhn.hapi.fhir:hapi-fhir-structures-r4:${versions.AppDependencies.Cql.hapiR4}"
```

Comment:
move this out of cql
```suggestion
    const val hapiFhirStructuresR4 = "ca.uhn.hapi.fhir:hapi-fhir-structures-r4:${versions.AppDependencies.Cql.hapiR4}"
```

Reply:
cf51117